app/src/main/java/com/example/android/undergraduatesattitude/Course.java

```diff
@@ -5,4 +5,79 @@
  */
 
 public class Course {
+  private ActivityDuration attendanceHours = new ActivityDuration();
```

Comment:
I think you should not instantiate the attendanceHours here. It should be done in the constructor.